csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -538,8 +538,38 @@ Secondly, the program fulfills the following for all items of:
 
 * type `/$defs/full_product_name_t/product_identification_helper/cpe`: If a CPE is invalid, the CSAF 2.0 to CSAF 2.1 converter SHOULD removed the
   invalid value and output a warning that an invalid CPE was detected and removed. Such a warning MUST include the invalid CPE.
+* type `/$defs/full_product_name_t/model_number`:
+  * If a model number is given that does not end on a star, the CSAF 2.0 to CSAF 2.1 converter SHOULD add a `*` to the end and output a
+    warning that a partial model number was detected and a star has been added.
+    Such a warning MUST include the model number.
+  * If the model number contains a `\`, the CSAF 2.0 to CSAF 2.1 converter MUST escape it by inserting an additional `\` before the character.
```

Comment:
Question also applies to serial identifier (a.k.a. number):
> Do we have a problem with UREL-encoding here if producers look up and insert such identifiers per web based systems that URL encode their payload? E.g. is "%5C%5c%5C*"
matching identifiers like "\\\*" (that is only this 4 character text) or like "%5C%5c%5-this-wun" and "%5C%5c%5-that-wun-too" and even "%5C%5c%5"?

I do not expect such corner cases to be significant or even advisable to use, but still maybe we should say something about this to avoid finger pointing between püarticipants of the eco system in the future?

Reply:
I'm not 100% sure whether I understood the case. If someone writes `%5C%5c%5C*` as a serial number
- serial numbers that match would be, for example:
```
%5C%5c%5C
%5C%5c%5C*
%5C%5c%5C1
%5C%5c%5CAVeryAnnoying Long Serial Number - With even Space and 10 whitespaces 
```

- serial numbers that don't match would be, for example:
```
%5C%5c%5
%5C%5c%5-this-wun
*1234
\*4711 
```

What am I missing?

Reply:
My corner case was that in browser editing environments (as secvisogram and others) the authors may be confused by copy-paste effects where the browser rewrites their data to make it conformant with RFCs and now verbatim backslashes intended as escaping meta characters end up becoming %5C or %5c and thus lose their escaping super power.

Reply:
Ah. I would suggest to leave that to the implementers for now (as I'm out of good ideas how to phrase that). But if you have a suggested wording, please open a PR.